db/manager.go

```diff
@@ -34,7 +35,9 @@ type Manager struct {
 
 	store   kt.TxnDatastoreExtended
 	network app.Net
-	dbs     map[thread.ID]*DB
+
+	lock sync.Mutex
```

Comment:
Maybe worth a `RWMutex` since most operations to `dbs` are reads?

Reply:
Yeah, was mostly a quick mutex for the hotfix.
This should be done in some other PR on the week targeting `master`, not this `v1.1.0-rc1` tag anyway.

Reply:
I won't merge this in master. I'll just use the commit hash to update the Hub to use this. (I'm not sure if `master` is really safe to use)